modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -338,7 +343,9 @@ object GmosControllerEpics extends GmosEncoders {
 
       private def nsParams(state: GmosNSEpicsState, config: NSConfig): List[IO[Unit]] =
         List(
-          applyParam(state.nsPairs, config.nsPairs, (x: Int) => DC.setNsPairs(x))
+          applyParam(state.nsPairs, config.nsPairs, (x: Int) => DC.setNsPairs(x)),
+          applyParam(state.nsRows, config.nsRows, (x: Int) => DC.setNsRows(x)),
```

Comment:
I would expect for this to not be necessary, because it will be set as part of the first N&S stage. In any case, the Tcl Seqexec does set it, so maybe it is needed.
The value set should correspond to the first stage.

Reply:
Fixed that and another bug that I just noted on the calculation of `nsPairs`